core/src/test/scala/org/apache/spark/scheduler/TaskSchedulerImplSuite.scala

```diff
@@ -140,6 +140,16 @@ class TaskSchedulerImplSuite extends SparkFunSuite with LocalSparkContext with B
     taskScheduler
   }
 
+  private def submitTasksAndCheck(
+      offers: IndexedSeq[WorkerOffer],
+      taskSets: Seq[TaskSet] = Seq(FakeTask.createTaskSet(1)),
+      taskSchedulerOpt: Option[TaskSchedulerImpl] = None) (f: Seq[TaskDescription] => Any): Any = {
```

Comment:
We can use the global `taskScheduler` as the default value?

Reply:
OK